decoder/DecoderApi_unittest.cpp

```diff
@@ -70,14 +84,42 @@ class TestDecodeFrames {
         return m_mime;
     }
 
+    const uint32_t getFrameCount() const
+    {
+        uint32_t i = 0;
+        FrameData d = m_data[i];
+        while(d.m_data != NULL) {
+            i++;
+            d = m_data[i];
+        }
+        return i;
+    }
+
     bool getFrame(VideoDecodeBuffer& buffer, FrameInfo& info)
     {
         const FrameData& d = m_data[m_idx];
         if (d.m_data != NULL) {
             buffer.data = (uint8_t*)d.m_data;
             buffer.size = d.m_size;
+            buffer.timeStamp = m_timestamp;
             info = d.m_info;
             m_idx++;
+            m_timestamp++;
+            return true;
+        }
+        return false;
+    }
+
+    bool getFrameByIndex(VideoDecodeBuffer& buffer, FrameInfo& info, int32_t index)
+    {
+        const FrameData& d = m_data[index];
+        if (d.m_data != NULL) {
+            buffer.data = (uint8_t*)d.m_data;
+            buffer.size = d.m_size;
+            buffer.timeStamp = m_timestamp;
```

Comment:
please do not put timestap thing in TestDecodeFrames, you can handle this in TEST_P(DecodeApiTest, Flush)